tutorials/reactingFoam/shearlayer_DLB/Allrun

```diff
@@ -1,10 +1,9 @@
 blockMesh
-N_PROCS=$(grep -c ^processor /proc/cpuinfo)
+N_PROCS=$(grep ^cpu\\scores /proc/cpuinfo | uniq |  awk '{print $4}')
```

Comment:
not sure, but perhaps nPROC/nThreads is more meaningful than nCores since we are mostly dealing with concurrent threads (SMT), so I would just revert to previous version?

Reply:
OpenFOAM does not benefit from concurrent threads (hyperthreading) as far as I know. Creating extra threads will only add extra overhead to manage them. So I would keep the change